.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,23 @@
+name: Style Check
+
+on:
+  pull_request:
```

Comment:
I change this so it runs on all PRs to be able to test this. I would revert this before merging.

Reply:
👌